### ISSUE TITLE
Add tags via email

### ISSUE THREAD

**Issue by arturtr** [ixti/redmine_tags#134]
Can I add tags to issue via email? Like I can add or update other fields.

Create via API works good, but how I can add tags via email?

Here is documentation about receiving email in redmine:
http://www.redmine.org/projects/redmine/wiki/RedmineReceivingEmails

Adding `tags` to `allow_override` param not works.
I try to add tags by adding this line to email:
```
Tags: hello
```

**Comment by sdwolf** [comment]
There is no such functionality AFAIK.
/cc @marius-balteanu should we include this for 3.2?

**Comment by marius-balteanu** [comment]
I don't think that is easy to implement because the file that parse the email and then make the request to Redmine is an external file (external means that the user must copy the file on the server where the emails are received). Only if we provide a new file with tags support but from my point of view, this feature has a very low priority. 

https://github.com/redmine/redmine/blob/master/extra/mail_handler/rdm-mailhandler.rb
